src/code42cli/util.py

```diff
@@ -173,3 +177,10 @@ def hash_event(event):
     if isinstance(event, dict):
         event = json.dumps(event, sort_keys=True)
     return md5(event.encode()).hexdigest()
+
+
+def parse_timestamp(date_str):
+    # example: {"property": "bar", "timestamp": "2020-11-23T17:13:26.239647Z"}
+    ts = date_str[:-1]
```

Comment:
borrowed from phcode42v2 and makes life easy

Reply:
really should just be using this everywhere, right? Not wanting to do that right now. Also, this PR makes me wish we were using the df formatter everywhere, we should try to prioritize that soon... And mock responses, either use `create_mock_response()` everywhere or your idea with generating response data...

Reply:
yeah, there have been a few things recently that I thought would be nice to just push into py42 to not have to rewrite everywhere..